great_expectations/data_context/store/profiler_store.py

```diff
@@ -24,7 +24,10 @@ def serialization_self_check(self, pretty_print: bool) -> None:
         """
         test_profiler_name = f"profiler_{''.join([random.choice(list('0123456789ABCDEF')) for _ in range(20)])}"
         test_profiler_configuration = RuleBasedProfilerConfig(
-            name=test_profiler_name, config_version=1.0, rules={}
+            name=test_profiler_name,
+            class_name="RuleBasedProfiler",
```

Comment:
@cdkini Assuming that it will "know" where/how to find the `module_name`, correct?..

Reply:
yes it should be fine! I've gone ahead and added it just to be more explicit